styles.css

```diff
@@ -9,13 +9,145 @@
     height: auto;
     list-style: none;
     box-sizing: border-box;
+    border: none;
   }
 
-  button {
-    border: none;
+  a {
+    color: #000;
+  }
+
+  /* header */
+
+  .header {
+    background-color: transparent;
+    position: fixed;
+    width: 100%;
+    z-index: 3;
+  }
+
+  .header ul {
+    margin: 0;
+    padding: 0;
+    list-style: none;
+    overflow: hidden;
+    background-color: #fff;
+  }
+
+  .header li a {
+    display: block;
+    padding: 20px 20px;
+    border-right: 1px solid #f4f4f4;
+    text-decoration: none;
+  }
+
+  .header .menu-btn {
+    display: none;
+  }
+
+  .header li a:hover,
+  .header .menu-btn:hover {
+    background-color: #f4f4f4;
+  }
+
+  .header .logo {
+    float: left;
+    font-size: 2em;
+    padding: 10px 20px;
+    text-decoration: none;
+  }
+
+  /* menu */
+
+  .header .menu {
+    clear: both;
+    max-height: 0;
+    transition: max-height 0.2s ease-out;
+  }
+
+  /* menu icon */
+
+  .header .menu-icon {
+    cursor: pointer;
+    display: inline-block;
+    float: right;
+    padding: 28px 20px;
+    position: relative;
+    user-select: none;
+  }
+
+  .header .menu-icon .navicon {
+    background: #333;
+    display: block;
+    height: 2px;
+    position: relative;
+    transition: background 0.2s ease-out;
+    width: 18px;
+  }
+
+  .header .menu-icon .navicon::before,
+  .header .menu-icon .navicon::after {
+    background: #333;
+    content: '';
+    display: block;
+    height: 100%;
+    position: absolute;
+    transition: all 0.2s ease-out;
+    width: 100%;
+  }
+
+  .header .menu-icon .navicon::before {
+    top: 5px;
+  }
+
+  .header .menu-icon .navicon::after {
+    top: -5px;
+  }
+
+  /* menu btn */
+  .header .menu-btn:checked ~ .menu {
+    max-height: 240px;
+  }
+
+  .header .menu-btn:checked ~ .menu-icon .navicon {
+    background: transparent;
+  }
+
+  .header .menu-btn:checked ~ .menu-icon .navicon::before {
+    transform: rotate(-45deg);
+  }
+
+  .header .menu-btn:checked ~ .menu-icon .navicon::after {
+    transform: rotate(45deg);
+  }
+
+  .header .menu-btn:checked ~ .menu-icon:not(.steps) .navicon::before,
+  .header .menu-btn:checked ~ .menu-icon:not(.steps) .navicon::after {
+    top: 0;
+  }
```

Comment:
- [ ] I can see you implemented the project functionality good job so far but there is something you are missing out as this project is there to help you learn more 
about :

1.  Basic JavaScript syntax.
2.  JavaScript to manipulate DOM elements.
3.  JavaScript events.
kindly consider implementing all the user's interactions required in the project requirement using javascript your js file so as to learn more about Js and DOM manipulation rather than implementing that using Css this will help you to meet all projects requirement and increase your level of understanding  on each point of the learning objectives section of the project